enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -0,0 +1,150 @@
+---
+title: CGroupsv2 Machine Config Pool Enablement
+authors:
+  - "@rphillips"
+reviewers:
+  - "@mrunalp"
+approvers:
+  - "@yuqi-zhang"
+  - "@mrunalp"
+creation-date: 2021-10-19
+last-updated: 2021-10-19
+status: implementable
+---
+
+# CgroupsV2 Machine Config Pool Enablement
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+cgroupsv2 enablement in Kubernetes has progressed to the point that it is
+usable. The underlying runtime (cri-o) and supporting subsystems are now ready
+for customers to begin their own testing with it. Not all workloads will be
+compatible with cgroupsv2, so it will *not* be enabled by default within
+Openshift at this time.
+
+## Motivation
+
+Migrating to Cgroupsv2 will bring in many new features and fixes not found in
```

Comment:
The [upstream kernel docs](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html) seem to use "cgroup v2".  The capitalization here is also inconsistent.  Let's both link to the upstream kernel docs, and consistently say "cgroup v2" to match?  (i.e. not "cgroupsv2" or "Cgroupsv2")

Reply:
I've updated the doc with a references and definitions section. good idea.